cmp/grid/Grid.ts

```diff
@@ -201,10 +213,10 @@ class GridLocalModel extends HoistModel {
                 clipboardCopy: Icon.copy({asHtml: true})
             },
             components: {
-                agColumnHeader: props => columnHeader({gridModel: model, ...props}),
-                agColumnGroupHeader: props => columnGroupHeader(props)
+                agColumnHeader: columnHeader,
```

Comment:
Is `gridModel` not a required prop of `ColumnHeader`? I see it referenced in `ColumnHeaderModel.gridModel`

Reply:
thanks for the careful look -- actually getting installed by agGrid through a special property on column:

 headerComponentParams: {gridModel, xhColumn: this},